[2022-05-22 03:30:45,914] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Teste_ETL_Raízen.convert_xls_in_xlsx scheduled__2022-05-22T02:30:00+00:00 [queued]>
[2022-05-22 03:30:45,925] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Teste_ETL_Raízen.convert_xls_in_xlsx scheduled__2022-05-22T02:30:00+00:00 [queued]>
[2022-05-22 03:30:45,925] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-05-22 03:30:45,925] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-05-22 03:30:45,925] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-05-22 03:30:45,940] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): convert_xls_in_xlsx> on 2022-05-22 02:30:00+00:00
[2022-05-22 03:30:45,945] {standard_task_runner.py:52} INFO - Started process 557 to run task
[2022-05-22 03:30:45,948] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Teste_ETL_Raízen', 'convert_xls_in_xlsx', 'scheduled__2022-05-22T02:30:00+00:00', '--job-id', '71', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Raizen.py', '--cfg-path', '/tmp/tmpddiv0ng2', '--error-file', '/tmp/tmpk5jz69rr']
[2022-05-22 03:30:45,949] {standard_task_runner.py:80} INFO - Job 71: Subtask convert_xls_in_xlsx
[2022-05-22 03:30:46,029] {task_command.py:369} INFO - Running <TaskInstance: Teste_ETL_Raízen.convert_xls_in_xlsx scheduled__2022-05-22T02:30:00+00:00 [running]> on host 541443bb6989
[2022-05-22 03:30:46,125] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Teste_ETL_Raízen
AIRFLOW_CTX_TASK_ID=convert_xls_in_xlsx
AIRFLOW_CTX_EXECUTION_DATE=2022-05-22T02:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-22T02:30:00+00:00
[2022-05-22 03:30:51,240] {python.py:173} INFO - Done. Returned value was: None
[2022-05-22 03:30:51,254] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Teste_ETL_Raízen, task_id=convert_xls_in_xlsx, execution_date=20220522T023000, start_date=20220522T033045, end_date=20220522T033051
[2022-05-22 03:30:51,295] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-22 03:30:51,329] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
